doc/install.txt

```diff
@@ -507,62 +507,38 @@ Note that:
-    **Solution 1:** make gcc-4.3 the default gcc
-    (http://pascalg.wordpress.com/2010/01/14/cuda-on-ubuntu-9-10linux-mint-helena/)
-    **Solution 2:** make another gcc (e.g. gcc-4.3) the default just for nvcc.
-    Do this by making a directory (e.g. ``$HOME/.theano/nvcc-bindir``) and
-    installing two symlinks in it: one called gcc pointing to gcc-4.3 (or lower) and one called
-    g++ pointing to g++-4.3 (or lower).  Then add
-    ``compiler_bindir = /path/to/nvcc-bindir`` to the ``[nvcc]`` section of your ``.theanorc``
-    (`libdoc_config`).
-
-
 .. _Graphviz: http://www.graphviz.org/Download_windows.php
 
 MacOS
 -----
 
-There are various ways to install Theano dependencies on a Mac.
-Here we describe the process in detail with EPD, Anaconda or with MacPorts, but if you did
-it differently and it worked, please let us know the details on the
-`theano-users`_ mailing-list, so that we can add alternate instructions
-here.
+There are various ways to install Theano dependencies on a Mac. Here
+we describe the process in detail with Canopy, Anaconda or Homebrew, but
+if you did it differently and it worked, please let us know the
+details on the `theano-users`_ mailing-list, so that we can add
+alternate instructions here.
 
 
-In academia: Enthought Python Distribution (EPD)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In academia: Enthought Canopy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you are working in academia, the easiest way to install most of the
 dependencies is to install
-`Enthought Python Distribution (EPD) <http://enthought.com/products/epd.php>`_.
-If you are affiliated with a university (as student or employee), you can
-download the installer for free.
+`Canopy <http://enthought.com/products/canopy/>`_.
+If you are affiliated with a university (as student or employee), you
+can download the installer for free.
 
-EPD installation includes in particular Python (and the development headers),
-NumPy, SciPy, nose, sphinx, easy_install, pydot (but *not* `Graphviz`_, which is
-necessary for it to work) and the MKL implementation of blas. The Mac OS and
-Linux version do not include g++.
+The Canopy installation includes in particular Python (and the
+development headers), NumPy, SciPy, nose, sphinx, pip, pydot
+(but *not* `Graphviz`_, which is necessary for it to work) and the MKL
+implementation of blas.
 
-pip is not included in EPD. After the installation of EPD,
-you can simply install it with::
-
-    .. code-block:: bash
-
-        $ sudo easy_install pip
-
-Then in a terminal execute this command to install the latest Theano release:
+To install the latest Theano release execute this in a terminal:
 
     .. code-block:: bash
 
         $ sudo pip install Theano
 
-If you want the bleeding edge version, `download
-and install git <http://git-scm.com/downloads>`_. Then in a
-terminal excute this command:
-
 .. code-block:: bash
 
     $ sudo pip install --upgrade --no-deps git+git://github.com/Theano/Theano.git
```

Comment:
We should at least keep "If you want the bleedeng edge version" before that command.